lib/ipmi_sdr.c

```diff
@@ -3159,7 +3160,8 @@ ipmi_sdr_get_record(struct ipmi_intf * intf, struct sdr_get_rs * header,
 			sdr_rq.length, sdr_rq.offset);
 
 		rsp = intf->sendrecv(intf, &req);
-		if (!rsp) {
+
+		if (!rsp || rsp->ccode == 0xca) {
```

Comment:
This is `IPMI_CC_CANT_RET_NUM_REQ_BYTES` from `ipmitool/ipmi_cc.h`. Please don't use magic numbers.

Reply:
I couldn't agree more (and I fixed it up), but to be fair, this is what the existing code did.